var/spack/repos/builtin/packages/liggghts/package.py

```diff
@@ -17,6 +17,7 @@ class Liggghts(MakefilePackage):
     git = "ssh://git@github.com/CFDEMproject/LIGGGHTS-PUBLIC.git"
 
     version("3.8.0", sha256="9cb2e6596f584463ac2f80e3ff7b9588b7e3638c44324635b6329df87b90ab03")
+    maintainers("SofiaXu")
```

Comment:
```suggestion
    maintainers("SofiaXu")
    
    version("3.8.0", sha256="9cb2e6596f584463ac2f80e3ff7b9588b7e3638c44324635b6329df87b90ab03")
```

@SofiaXu thanks for the fix! Just a quick suggestion, often times maintainers are listed before versions to make it a bit easier to find the maintainer in packages with lots of versions.